Tasks/PublishBuildArtifactsV1/task.json

```diff
@@ -75,6 +75,15 @@
             "required": false,
             "helpMarkDown": "Enter the degree of parallelism, or number of threads used, to perform the copy. The value must be at least 1 and not greater than 128.",
             "visibleRule": "ArtifactType = FilePath && Parallel = true"
+        },
+        {
+            "name": "FileCopyOptions",
+            "type": "string",
+            "label": "File copy options",
+            "defaultValue": "",
+            "required": false,
+            "helpMarkDown": "Pass additional options to the file copy command.",
```

Comment:
will this always be robocopy?  Would be good to specify if possible so people know what they can pass

Reply:
Yep, it's always `robocopy`, so I've updated the description